lib/TPP/ConvertLinalgToTpp.cpp

```diff
@@ -35,31 +35,55 @@ struct ConvertGenericOpToTpp : public OpRewritePattern<linalg::GenericOp> {
   LogicalResult rewriteToTppOp(linalg::GenericOp linalgOp,
                                PatternRewriter &rewriter) const {
     SmallVector<Value> operands;
+    if (tpp::utils::isTppZero(linalgOp, &operands)) {
+      assert((operands.size() != 0 && operands.size() <= 2) &&
+             "zero expects one or two operands");
```

Comment:
this comment trips my mind a bit: "zero expects one or two" -> "tpp.zero op expects.."